### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by malte-v**
Someone on Reddit had trouble building the PDFs, so now people who aren't familiar with LaTeX can just download per-commit versions instead of using the outdated ones from the course page.

### CODE REVIEW

**Comment by favonia** [comment]
Thank you! This is a cool idea. I am a bit worried that GitHub will clean up the artifacts after 90 days of inactivity, though. (At least that's what the documentation says.) Perhaps we should run the action monthly as well, maybe the first day of every month? What do you think about it? I have not thought through this carefully.

By the way, I might also add a link to https://docs.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#downloading-or-deleting-artifacts in README for people who are not familiar with artifacts. Another thing is that `make` _should_ have been installed already in the Ubuntu images, but I guess it does no harm to reinstall it during CI.

**Comment by malte-v** [comment]
> GitHub will clean up the artifacts after 90 days of inactivity

Oops, thanks for pointing that out!

I've configured the CI to run on the 1st of every month, removed the unnecessary `make` dependency and added your link to the README.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# [15-819 (Homotopy Type Theory)](http://www.cs.cmu.edu/~rwh/courses/hott/) Lecture Notes
+
+Per-commit PDFs can be downloaded from [GitHub Actions](https://github.com/RobertHarper/hott-notes/actions). For detailed instructions, see here: https://docs.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#downloading-or-deleting-artifacts
```
Comment (favonia) [inline README.md:3]: How about this?
```suggestion
Per-commit PDFs can be downloaded from [GitHub Actions](https://github.com/RobertHarper/hott-notes/actions). See the [detailed instructions for downloading artifacts](https://docs.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#downloading-or-deleting-artifacts)
```

**Comment by favonia** [comment]
Thank you again for your work. Let's see if the CI will be enabled automatically...

**Pull request merged**